src/commands/ml-models-delete.ts

```diff
@@ -0,0 +1,9 @@
+import { Command } from "../command";
+
+module.exports = new Command("ml:models:delete <modelId>")
```

Comment:
prefer `export default` in this case
```suggestion
export default new Command("ml:models:delete <modelId>")
```